File: python/Course.py
class Course:
    # Inisialisasi variabel untuk nama matakuliah
    _nama_matakuliah = ""

    # Konstruktor untuk menginisialisasi variabel nama matakuliah
    def __init__(self, nama_matakuliah=''):
        self._nama_matakuliah = nama_matakuliah

    # Getter untuk mendapatkan nilai nama matakuliah
    def getNamaMatakuliah(self):
        return self._nama_matakuliah

File: cpp/Course.cpp
// // import library
// #include <iostream>
// #include <string>

// using namespace std;

class Course{
    protected:
        string namaMatakuliah;
    public:
        Course(string namaMatakuliah = ""){
            this->namaMatakuliah = namaMatakuliah;
        }

        string getNamaMatakuliah(){return namaMatakuliah;}

        ~Course(){}
};